src/unum/util/util_zlib.h

```diff
@@ -0,0 +1,26 @@
+// (c) 2020 minim.co
+// unum zlib utils include file
+
+#ifndef _UTIL_ZLIB_H
+#define _UTIL_ZLIB_H
+#ifdef FEATURE_GZIP // Only if this is feature is enabled
+
+#include <zlib.h>
+
+
+// Threshold beyond which the message is to be compressed
+// Default value is 0
+// Can be overwritten by platform specific defines
```

Comment:
Let's remove this line, platforms should start the agent with the option or config file setting the non-zero threshold.